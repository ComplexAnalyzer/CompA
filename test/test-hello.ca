int main()
{
  println(42);
  println(71.3);
  println((3.2,3.4));
  println("hello world");

  print(42);
  print(71.3);
  print((3.2,3.4));
  print("hello world");
  return 0;
}
