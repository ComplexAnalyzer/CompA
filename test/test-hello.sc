def main() 
{
prints("hello world");
}
