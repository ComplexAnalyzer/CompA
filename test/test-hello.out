42
71.300000
(3.200000,3.400000)
hello world
4271.300000(3.200000,3.400000)hello world